### PR TITLE
Remove iframe code in automation testing

### PR DESCRIPTION
**Pull request by makambalaji**
### What does this PR do?
#21255 

### What issues does this PR fix or reference?

Removed iframe code from javaPlugin, pythonPlugin and DevWorkSpaceHappyPath sepc files and deprecated the iframe related functions. 

### How to test this PR?

**Test Platform used**: minikube
**Execution Commands:**
```
export TS_SELENIUM_BASE_URL=https://192.168.59.179.nip.io
export TS_SELENIUM_USERNAME=che@eclipse.org
export TS_SELENIUM_PASSWORD=admin
export TS_SELENIUM_LOG_LEVEL=DEBUG
export USERSTORY=JavaPlugin or PythonPlugin
npm run test-plugin
-------------------------------------------------------------------
For DevWorkspaceHappyPath Spec file Execution
--------------------------------------------------------------------
export TS_SELENIUM_BASE_URL=https://192.168.59.179.nip.io
export TS_SELENIUM_USERNAME=che@eclipse.org
export TS_SELENIUM_PASSWORD=admin
export TS_SELENIUM_DEVWORKSPACE_URL=https://192.168.59.179.nip.io/#https://github.com/che-samples/java-spring-petclinic/tree/devfilev2
export TS_SELENIUM_LOG_LEVEL=DEBUG
npm run test-devworkspace-happy-path
```

**Execution Logs**: 
[PythonPlugin_ExecLogs.txt](https://github.com/eclipse/che/files/8260014/PythonPlugin_ExecLogs.txt)
[JavaPlugin_ExecLogs.txt](https://github.com/eclipse/che/files/8260126/JavaPlugin_ExecLogs.txt)
[DevWorkspace_HappyPath_ExecLogs.txt](https://github.com/eclipse/che/files/8260840/DevWorkspace_HappyPath_ExecLogs.txt)

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 1]
```diff
@@ -29,7 +29,8 @@ export enum LeftToolbarButton {
 export class Ide {
     public static readonly EXPLORER_BUTTON_ID: string = 'shell-tab-explorer-view-container';
     public static readonly SELECTED_EXPLORER_BUTTON_CSS: string = 'li#shell-tab-explorer-view-container.theia-mod-active';
-    public static readonly ACTIVATED_IDE_IFRAME_CSS: string = '#ide-iframe-window[aria-hidden=\'false\']';
+    public static readonly THEIA_EDITOR_CSS: string = '#theia-main-content-panel';
+    // public static readonly ACTIVATED_IDE_IFRAME_CSS: string = '#ide-iframe-window[aria-hidden=\'false\']';
```
Comment (SkorikSergey) [inline tests/e2e/pageobjects/ide/Ide.ts]: Let's remove `ACTIVATED_IDE_IFRAME_CSS` if we don't need it anymore.

**Review by SkorikSergey — APPROVED** [review]

File: tests/e2e/pageobjects/ide/Ide.ts (hunk 1 above)
Comment (dmytro-ndp) [reply tests/e2e/pageobjects/ide/Ide.ts]: let's remove commented code to keep script clean

File: tests/e2e/pageobjects/ide/Ide.ts [hunk 2]
```diff
@@ -72,6 +77,15 @@ export class Ide {
...
 
+    async waitForEditor(timeout: number = TimeoutConstants.TS_SELENIUM_START_WORKSPACE_TIMEOUT) {
+        Logger.debug('Ide.waitForEditor');
+        try {
+            await this.driverHelper.waitVisibility(By.css(Ide.THEIA_EDITOR_CSS), timeout);
+        } catch (err) {
+            Logger.warn(`Editor is not displayed even after ${timeout} milliseconds.`);
+            throw err;
+        }
+    }
```
Comment (dmytro-ndp) [inline tests/e2e/pageobjects/ide/Ide.ts:87]: please, divide methods by empty lines to improve readability

File: tests/e2e/testsLibrary/LanguageServerTests.ts [hunk 3]
```diff
@@ -164,9 +162,10 @@ export class LanguageServerTests {
             } catch (err) {
                 // debug config is probably missing, refresh IDE and try again https://github.com/eclipse/che/issues/19887
                 await this.browserTabsUtil.refreshPage();
-                await this.driverHelper.wait(TimeoutConstants.TS_IDE_LOAD_TIMEOUT);
-                await this.ide.waitAndSwitchToIdeFrame();
-                await this.driverHelper.wait(TimeoutConstants.TS_SELENIUM_CLICK_ON_VISIBLE_ITEM);
+                await this.ide.waitIde();
```
Comment (dmytro-ndp) [inline tests/e2e/testsLibrary/LanguageServerTests.ts:165]: let's remove commented code to keep script clean

File: tests/e2e/testsLibrary/LanguageServerTests.ts [hunk 4]
```diff
@@ -187,9 +186,10 @@ export class LanguageServerTests {
...
+                // await this.driverHelper.wait(TimeoutConstants.TS_IDE_LOAD_TIMEOUT);
```
Comment (dmytro-ndp) [inline tests/e2e/testsLibrary/LanguageServerTests.ts]: let's remove commented code to keep script clean

File: tests/e2e/testsLibrary/LanguageServerTests.ts (hunk 3 above)
Comment (makambalaji) [reply tests/e2e/testsLibrary/LanguageServerTests.ts:165]: I will remove the commented lines

Comment (dmytro-ndp) [reply tests/e2e/testsLibrary/LanguageServerTests.ts:165]: Thanks.

**Review by dmytro-ndp — APPROVED** [review]
LGTM

**Pull request merged**